scrapers/pornworld.yml

```diff
@@ -1,24 +1,40 @@
-name: pornworld
+name: Pornworld
 sceneByURL:
   - action: scrapeXPath
     url:
       - pornworld.com/watch/
     scraper: sceneScraper
+sceneByFragment:
+  action: scrapeJson
+  queryURL: https://pornworld.com/autocomplete?query={filename}
+  queryURLReplace:
+    filename:
+      - regex: .+(GP\d+).*
+        with: $1
+  scraper: sceneQueryScraper
+jsonScrapers:
+  sceneQueryScraper:
+    scene:
+      Title: terms.Scene.0.name
+      URL: terms.Scene.0.url
 xPathScrapers:
   sceneScraper:
     scene:
-      Title: //meta[@name="twitter:title"]/@content
-      Date: //span[@title="Release date"]/a/text()
-      Details: //div[@class="scene-description__row"]/dt[contains(.,"Description")]/../dd/text()
+      Title: 
+        selector: //h1
+        postProcess:
+          - replace:
+            - regex: featuring.+
+              with: ""
+      Date: //i[contains(@class,"calendar3")]/text()
+      Details: //meta[@name="description"]/@content
```

Comment:
In some scenes there is a `weird` char when you scrape eg `https://pornworld.com/watch/207450`
A replace is needed to clean that up
```yaml
      Details:
        selector: //meta[@name="description"]/@content
        postProcess:
          - replace:
              - regex: '\x{0092}' # unicode char 0092 (looks like ')
                with: "'"
```